examples/notebooks/remoteRover.ipynb

```diff
@@ -15,10 +15,12 @@
    "metadata": {},
    "source": [
     "#!rover\n",
-    "resourceScanner.CaptureFromCamera(roverBody.Camera);\n",
+    "Microsoft.DotNet.Interactive.Formatting.Formatter.ListExpansionLimit = 42;\n",
+    "ResourceScanner.CaptureFromCamera(RoverBody.Camera);\n",
+    "ResourceScanner.UseUri(new Uri(\"https://lobe-diego.ngrok.io/predict/34bc1074-f2fd-48bd-b473-7d1c4b50cd34\"));\n",
+    "//ResourceScanner.UseModel(new DirectoryInfo(\"/home/pi/dotnet_conf_resources/models/v5.3\"));\n",
     "\n",
-    "//resourceScanner.UseModel(new DirectoryInfo(\"/home/pi/dotnet_conf_resources/models/v5.3\"));\n",
-    "resourceScanner.UseUri(new Uri(\"https://lobe-diego.ngrok.io/predict/34bc1074-f2fd-48bd-b473-7d1c4b50cd34\"));"
+    "RoverBody.Reset();"
```

Comment:
As I understand, this `RoverBody.Reset()` is required to safely call the RoverBrain methods? They don't get their default value (() => {}), until you call reset to set them to null, right?

I think it's odd for this to be part of the user api. Maybe we should call RoverBody.Reset the RoverRobotSetup to ensure those methods are defined? Or put the defaults on the getters instead of the setters?

Reply:
So, until you call Reset the loop in RoverRobotSetup falls into the catch blocks and logs errors like this:
```
2020-11-09T21:09:06.8786918Z [React]  ❌ (79.1962ms) System.NullReferenceException: Object reference not set to an instance of an object.
   at PiTop.Interactive.Rover.RoverRobotSetup.<>c__DisplayClass1_0.<ConfigureRover>b__1() in /home/pi/pi-top-4-.NET-Core-API/src/Examples/PiTop.Interactive.Rover/RoverRobotSetup.cs:line 123 System.NullReferenceException: Object reference not set to an instance of an object.
   at PiTop.Interactive.Rover.RoverRobotSetup.<>c__DisplayClass1_0.<ConfigureRover>b__1() in /home/pi/pi-top-4-.NET-Core-API/src/Examples/PiTop.Interactive.Rover/RoverRobotSetup.cs:line 123
```

Reply:
fixed